app/commands/github/pull_request/sync_repo.rb

```diff
@@ -0,0 +1,143 @@
+module Github
+  class PullRequest
+    class SyncRepo
+      include Mandate
+
+      initialize_with :repo
+
+      def call
+        pull_requests.each do |pr|
+          Github::PullRequest::CreateOrUpdate.(
+            pr[:node_id],
```

Comment:
Do you think I makes sense to have just the node_id as a regular argument, instead of also passing it as a named argument?